packages/website/src/components/typeDetails/TypesDetails.tsx

```diff
@@ -51,17 +51,15 @@ export function TypesDetails({
         </div>
       </Panel>
       <PanelResizeHandle className={styles.PanelResizeHandle} />
-      {selectedNode && (
```

Comment:
i *vaguely* remember armano saying this was a necessary condition?
might be worth double checking the preview deploy

Reply:
Works locally and on the preview for me 🚀